File: euler_001.py
from time import time

start = time()

mult_3_and_5 = set([n for n in range(1000) if n%3 == 0 or n%5 == 0])

print("Sum is " + str(sum(mult_3_and_5)))

stop = time()

print("Time: " + str(stop-start) + " secs")

File: euler_001.js
const start = Date.now();

let arr = Array(1001);
arr.fill(false);

let sum = 0;

let i = 1;
while (i * 3 < 1000) {
    if (!arr[3 * i]) {
        arr[3 * i] = true;
        sum += 3 * i;
    }
    i++;
}
let j = 1;
while (j * 5 < 1000) {
    if (!arr[5 * j]) {
        arr[5 * j] = true;
        sum += 5 * j;
    }
    j++;
}


const end = Date.now();

console.log("Execute time: ", end - start, " Sum is: ", sum);